### DISCUSSION TITLE
cgroup1 and ha

### DISCUSSION THREAD

**Discussion by a-dasler** [devmfc/debian-on-amlogic#5] [Q&A] [answered]
Hello, I encountered this problem
When installing Home Assistant on tvbox I get the following error:

Currently Supervisor requires CGroup v1 in a supervised installation.

Armbian describes a fix method, but we have a different loading method, as I understand it...
I can't figure out where to paste:

systemd.unified_cgroup_hierarchy=0

I have a system and boot from emmc.

**Comment by devmfc** [comment, marked as answer]
Add the following line in _/boot/boot.config_:

`bootargs0=systemd.unified_cgroup_hierarchy=0`

This will add the parameter at the end of the bootargs variable which in turn will become the kernel's command-line parameters.

After booting you can check with `cat /proc/cmdline`

Good luck!

**Comment by a-dasler** [comment]
It worked, great!!!
No error!!
